src/app/redux/tests/global.test.js

```diff
@@ -3,11 +3,9 @@
 import chai, {expect} from 'chai';
 import chaiImmutable from 'chai-immutable';
 import Immutable, {Map} from 'immutable';
-import ReducerModule from '../GlobalReducer';
+import reducer, * as actions from '../GlobalReducer';
```

Comment:
`* as globalActions` would be better, for consistency.

Reply:
eh it's a test

Reply:
That's just sloppy.